hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveCatalog.java

```diff
@@ -354,7 +402,7 @@ public void testCreateTableCustomSortOrder() throws Exception {
   }
 
   @Test
-  public void testCreateNamespace() throws Exception {
+  public void testDatabaseAndNamespaceWithLocation() throws Exception {
```

Comment:
Base class test [testCreateNamespace](https://github.com/apache/iceberg/blob/385a6bd296e551bdfdab48d2a291d3516f999f5a/core/src/test/java/org/apache/iceberg/catalog/CatalogTests.java#L167)  is different from the current one.